elementary/monitor/dbt_project_utils.py

```diff
@@ -1,15 +1,62 @@
 import os
+from typing import Optional
+
+from elementary.utils.ordered_yaml import OrderedYaml
 
 _MONITOR_DIR = os.path.dirname(os.path.realpath(__file__))
 
 _DBT_PACKAGE_NAME = "elementary"
+_DBT_PROJECT_FILENAME = "dbt_project.yml"
+_PACKAGES_FILENAME = "packages.yml"
 
 PATH = os.path.join(_MONITOR_DIR, "dbt_project")
 
 # Compatibility for previous dbt versions
-_MODULES_PATH = os.path.join(PATH, "dbt_modules", _DBT_PACKAGE_NAME)
-_PACKAGES_PATH = os.path.join(PATH, "dbt_packages", _DBT_PACKAGE_NAME)
+_MODULES_PATH = os.path.join(PATH, "dbt_modules")
+_PACKAGES_PATH = os.path.join(PATH, "dbt_packages")
+
+
+def get_elementary_package_path():
+    package_path = os.path.join(_PACKAGES_PATH, _DBT_PACKAGE_NAME)
+    if os.path.exists(package_path):
+        return package_path
+
+    legacy_package_path = os.path.join(_PACKAGES_PATH, _DBT_PACKAGE_NAME)
+    if os.path.exists(legacy_package_path):
+        return legacy_package_path
+
+    return None
+
+
+def dbt_package_up_to_date() -> bool:
```

Comment:
1. Just for extra clarity, append with `is_`?
2. Do you think we should make that validation generic for the `packages.yml` so that it'll also include the `dbt-utils` or other packages we might add in the future? Or is it an overhead for the time being?

Reply:
I'll change(1)
re. (2( - I actually thought the same and started with implementing it like this, but then saw that annoyingly the version inside `dbt-project.yml` for `dbt-utils` was not consistent with the version that was installed :( :(

See this for example:
https://github.com/dbt-labs/dbt-utils/blob/0.9.6/dbt_project.yml

So changed it for now to be only on our package. I could also blacklist `dbt-utils` but it felt a bit pointless as it's the only package aside from our own :)

Reply:
Ohh right.
Totally reasonable then.
We’ll have to be very on point about bumping the version ourselves.
Let’s merge!